levenshtein/migrations/0001_initial.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'Levenshtein'
+        db.create_table(u'levenshtein_levenshtein', (
+            (u'id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('text_1', self.gf('django.db.models.fields.TextField')()),
+            ('text_2', self.gf('django.db.models.fields.TextField')()),
+            ('levenshtein_value', self.gf('django.db.models.fields.IntegerField')(null=True, blank=True)),
```

Comment:
Tip: Your model already is called `Levenshtein`. No need to repeat that in the field names. `levenshtein_value` could be called `distance`, for example.